src/styles/page.scss

```
@@ -0,0 +1,19 @@
+.page__catalog {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 50px auto;
+  width: ($cardWidth + $gap) * 4;
+  gap: $gap;
+}
+
+.page__header {
+  position: sticky;
```

you can use scss nesting here
```suggestion
.page {
  &__catalog {...}
  &__header {...}
}
```